src/python/pants/releases/packages.py

```diff
@@ -134,6 +146,51 @@ def all_packages() -> Set[Package]:
     return core_packages().union(contrib_packages())
 
 
+# -----------------------------------------------------------------------------------------------
+# Script utils
+# -----------------------------------------------------------------------------------------------
+
+
+def get_pypi_config(section: str, option: str) -> str:
+    config = ConfigParser()
+    config.read(os.path.expanduser("~/.pypirc"))
+    if not config.has_option(section, option):
+        raise ValueError(f"Your ~/.pypirc must define a {option} option in the {section} section")
+    return config.get(section, option)
+
+
+def get_git_branch() -> str:
+    return (
+        subprocess.run(
+            ["git", "rev-parse", "--abbrev-ref", "HEAD"], stdout=subprocess.PIPE, check=True
+        )
+        .stdout.decode()
+        .strip()
+    )
+
+
+def get_pgp_key_id() -> str:
+    return (
+        subprocess.run(["git", "config", "--get", "user.signingkey"], stdout=subprocess.PIPE)
```

Comment:
Should this use `check=True`? And if so, should these use `subprocess.check_output`?

Reply:
I originally had `check=True`, but it resulted in the script crashing if the values aren't configured, whereas we want to gracefully handle that situation.